kformat/exception.py

```diff
@@ -31,3 +31,12 @@ def __init__(self, length: int):
 
     def __str__(self) -> str:
         return f'Invalid length of value is given(expected: {self.length})'
+
+
+class UnicodeKFormatError(KFormatError, UnicodeError):
```

Comment:
```suggestion
class UnsupportedUnicodeError(KFormatError, UnicodeError):
```

정도의 이름은 어떠신가요? K Format이 꼭 이름에 들어갈 필요는 없을 것 같고 K Format에서 지원하지 않는 유니코드 정도로 커뮤니케이션하는 게 좋을 것 같습니다 😃

Reply:
좋습니다. 변경 했습니다!